### PR TITLE
reword of sessions types and durations

### PR DESCRIPTION
**Pull request by mpetitdant**
A few wording to review

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -327,7 +327,7 @@ talk.descriptive.section=Descriptif
 talk.title=Titre
 talk.title.note=Obligatoire. Un court titre pour votre session
 talk.format=Format
-talk.format.note=Obligatoire.
+talk.format.note=Obligatoire. Une keynote "alien" est une session dans le grand amphi avec tout le monde sur un sujet ni technique ni alien mais ayant un lien avec nos métiers.
```
Comment (clacote) [inline conf/messages]: "ni technique ni alien" -> "ni technique ni agile/méthodologique"

File: conf/messages.en [hunk 2]
```diff
@@ -301,7 +301,7 @@ talk.ideaForNow.show=Ideas for now
 talk.title=Title
 talk.title.note=Required. A short title for your talk
 talk.format=Format
-talk.format.note=Required.
+talk.format.note=Required. An "alien" keynote is a session in our large room with everybody on a non tachnical nor agile subject but with a small link with our jobs.
```
Comment (clacote) [inline conf/messages.en]: tachnical -> technical
with everybody -> in front of all attendees.
with a small link to our jobs -> with a connection to our jobs.

File: conf/messages (hunk 1 above)
Comment (mpetitdant) [inline conf/messages]: ça sent la fatigue :-)

**Comment by mpetitdant** [comment]
Better ?

**Comment by clacote** [comment]
:+1:

**Pull request merged**